packages/documentation/src/stories/components/forms/validation/validation.stories.ts

```diff
@@ -0,0 +1,298 @@
+import { Args, StoryObj } from '@storybook/web-components';
+import { html, nothing } from 'lit';
+import { MetaComponent } from '@root/types';
+
+const meta: MetaComponent = {
+  id: '1aa900d9-aa65-4ae0-b8cd-e6cca6cc3472',
+  title: 'Components/Forms/Validation',
+  tags: ['package:HTML'],
+  parameters: {
+    badges: [],
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/design/JIT5AdGYqv6bDRpfBPV8XR/Foundations-%26-Components-Next-Level?node-id=577-14513&t=uKEtTo9BEaPpKSV5-1',
+    },
+  },
+};
+
+const validationObject: object = {
+  name: 'Validation',
+  description: 'Controls the validation state appearance of the component.',
+  control: {
+    type: 'radio',
+    labels: {
+      'null': 'Default',
+      'is-valid': 'Valid',
+      'is-invalid': 'Invalid',
+    },
+  },
+  options: ['null', 'is-valid', 'is-invalid'],
+  table: {
+    category: 'States',
+  },
+};
+
+export default meta;
+
+type Story = StoryObj;
+export const Default: Story = {};
+
+export const Checkbox: Story = {
+  args: {
+    validation2: 'is-invalid',
+  },
+  argTypes: {
+    validation2: validationObject,
+  },
+  render(args: Args) {
+    const isValidationSet = args.validation2 !== 'null';
+    const isValid = args.validation2 === 'is-valid';
+    let ariaInvalid;
+    if (isValidationSet) {
+      ariaInvalid = !isValid;
+    } else {
+      ariaInvalid = nothing;
+    }
```

Comment:
Why not using a ternary operator here: 
```suggestion
    const ariaInvalid = isValidationSet ? !isValid : nothing;
```